subt_ign/launch/competition.ign

```diff
@@ -247,6 +247,11 @@
     <value>$LD_LIBRARY_PATH</value>
   </env>
 
+  <env>
```

Comment:
nit: I'm not sure why this launch file has this. Shouldn't it be just in cloudsim_sim.ign?

Reply:
I put this here so that the behavior is the same when running locally using this launch file and running in the cloud.